integration-tests/jira/pom.xml

```diff
@@ -94,6 +94,16 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-failsafe-plugin</artifactId>
+                        <configuration>
+                            <!-- Theses classpath configurations must be removed when https://ecosystem.atlassian.net/browse/JRJC-262 gets resolved-->
+                            <additionalClasspathElements>
+                                <additionalClasspathElement>${project.build.directory}/camel-quarkus-integration-test-jira-${project.version}-native-image-source-jar/lib/modified-com.atlassian.jira.jira-rest-java-client-core-5.2.4.jar</additionalClasspathElement>
```

Comment:
This will fail on `camel-main` as the client has been upgraded in Camel to 5.2.5.

Can you add a version property for this into the project root pom.xml. E.g

```
<jira-rest-client.version>${jira-rest-client-api-version}</jira-rest-client.version>
```

Reply:
Good point - I will add it!